tests/pulseaudio_dbus_spec.lua

```diff
@@ -189,4 +190,17 @@ b.describe("PulseAudio with DBus", function ()
                 	end
 		   		end
            end)
+
+		   b.it("Will set the next sink as the FallbackSink", function()
+		   	   if total_number_of_sinks <= 1 then
+				   print("\nWARNING: won't set the next sink as the FallbackSink because there is only one sink available in this machine")
```

Comment:
This should print a message saying that the test will be skipped rather than a more worrying warning.